py-rattler/tests/unit/test_solver.py

```diff
@@ -0,0 +1,108 @@
+# type: ignore
+import os.path
+import subprocess
+
+import pytest
+from rattler import (
+    Channel,
+    ChannelConfig,
+    fetch_repo_data,
+    Platform,
+    solve,
+    MatchSpec,
+    RepoDataRecord,
+)
+
+
+@pytest.fixture(scope="session")
+def noarch_repo_data() -> None:
```

Comment:
Maybe it would be nicer to return a `SparseRepoData` from a file here instead of spinning up a server. I think that would speed up the tests.